InterfaceStubGenerator.Core/InterfaceStubGenerator.cs

```diff
@@ -169,9 +169,20 @@ public TemplateInformation GenerateTemplateInfoForInterfaceList(List<InterfaceDe
                 UsingList = usings.ToList()
             };
 
+            AddInheritedMethods(ret);
+
             return ret;
         }
 
+        private static void AddInheritedMethods(TemplateInformation ret)
+        {
+            foreach (var c in ret.ClassList.Where(c => c.BaseClassNames != null && c.BaseClassNames.Any()))
```

Comment:
Does this logic handle two interfaces with the same name but in different namespaces?